### ISSUE TITLE
[Design] HTML email templates (v2)

### ISSUE THREAD

**Issue by reitermb** [raft-tech/TANF-app#1708]
**Description:**
Delivers compliant HTML email templates and documentation regarding email triggers for V2 notification events

**AC:**

- [x] Documentation work for the following has occurred:
   - [x] Notification triggers
- [x] The design is usable, meaning it adheres to definition of done standards for project design work.

> <details>
>      <summary>Expand DoD Standards</summary>
> 
> - It uses [USWDS components and follows it’s UX guidance](https://designsystem.digital.gov/components/), or a deviation is clearly documented
> 
> - Language is intentional and [plain](https://plainlanguage.gov/guidelines/); placeholders are clearly documented
> - It follows [accessibility guidelines] and accessibility implementation notes are documented(https://accessibility.digital.gov/) (e.g. clear information hierarchy, color is not the only way meaning is communicated, etc.)
> - If feedback identifies bigger questions or unknowns, create additional issues to investigate
> </details>
> 

- [ ] Dev/Design sync has occurred; resulting tickets created.

**Tasks:**
- [x] Obtain final logo to use in the email template
- [x] Update notification content as needed (e.g. FAQ placeholder links should be updated to point toward the knowledge center)
- [x] Update notification trigger documentation so that deactivation warning email is sent out 10 days, 3 days and 1 day
- [x] Create HTML email templates
   - [x] 1814 - As a TDP User, I want to be notified when my request for access to TDP has been submitted
   - [x] 1839 - As a TDP User, I want to be notified when my request for access to TDP has been approved
   - [x]  1334 - As a person who has been denied access, I want to receive an email notifying me that my request has been denied
   - [x] 1844 - As a TDP User, I want to be notified when my account will soon be deactivated
   - [x] 1840 - As a TDP User, I want to be notified when my TDP account is deactivated
   - [x] 1845 - As an STT analyst, I want to be notified when data for my program has been submitted
- [x] Send test emails using templates, evaluate conformance to HTML email standards, accessibility.
- [x] Run emails through accessibility insights for web through GitHub
- [ ] Logo image will be hosted in S3 bucket - Andrew to provide links

**Supporting Documentation:**
- [hack.md](https://hackmd.io/L0uwkVgxSPqfLBwOcuozPg)
- [Figma] (https://www.figma.com/file/irgQPLTrajxCXNiYBTEnMV/TDP-Mockups-For-Feedback?node-id=7196%3A50118)
- [Directory of Emails](https://reitermb.github.io/TDP-HTML-Email-Demo-Environment/)

**Comment by reitermb** [comment]
Took out triggers from this email because we believe it fits well in #1462 / remainder of the sprint

**Comment by lilybl1** [comment]
Will review the copy suggestions and question in Monday's UX Sync before continuing any further. The current email copy based on the HackMD and can be found here https://reitermb.github.io/TDP-HTML-Email-Demo-Environment/

**Comment by lilybl1** [comment]
@lfrohlich https://www.figma.com/file/irgQPLTrajxCXNiYBTEnMV/TDP-Mockups-For-Feedback?node-id=7196%3A50118

**Comment by lilybl1** [comment]
Per the UX Sync meeting:

**Comment by lilybl1** [comment]
@lfrohlich @ADPennington @stevenino this is ready for QASP - you can preview the final copy changes here: https://reitermb.github.io/TDP-HTML-Email-Demo-Environment/

**Comment by reitermb** [comment]
@ttran-hub these emails should also get a gov a11y review pass. They're testable on web at the link @lilybl1 provided above, but should also get a pass once they've actually been sent to an email client since that's the medium it'll ultimately be displayed in. The raw HTML ([accessible in the repo](https://github.com/reitermb/TDP-HTML-Email-Demo-Environment)) can be pasted into any HTML email delivery service—we've been using https://putsmail.com/ as a simple & free option in our testing.

**Comment by lfrohlich** [comment]
@lilybl1 content is looking good! I made a few minor comments in the figma. can you also add the figma link to the description of the ticket?

**Comment by ADPennington** [comment]
> @ttran-hub these emails should also get a gov a11y review pass. They're testable on web at the link @lilybl1 provided above, but should also get a pass once they've actually been sent to an email client since that's the medium it'll ultimately be displayed in. The raw HTML ([accessible in the repo](https://github.com/reitermb/TDP-HTML-Email-Demo-Environment)) can be pasted into any HTML email delivery service—we've been using https://putsmail.com/ as a simple & free option in our testing.

@ttran-hub see the links 🆙

**Comment by lilybl1** [comment]
@lfrohlich The edits in Figma have been completed and updated in HackMD as well. Thanks!

**Comment by ttran-hub** [comment]
> https://reitermb.github.io/TDP-HTML-Email-Demo-Environment/

FYI: 404 error link to the Getting Started guide in the content of the template. 
Otherwise, the email templates are good with a11y. 🚀

**Comment by lfrohlich** [comment]
With a11y review, this is approved. Thanks!
